cognite_toolkit/_cdf_tk/builders/_file.py

```diff
@@ -29,7 +30,7 @@ def build(
                 if warning is not None:
                     yield [warning]
                 continue
-            if loader is FileMetadataLoader:
+            if loader in {FileMetadataLoader, CogniteFileLoader}:
```

Comment:
This is what adds the feature.

Reply:
Curious: why not [FileMetadataLoader, CogniteFileLoader]?

Reply:
O(1) instead of O(2). However, a set has more overhead, so using a list is probably faster.